source/common/runtime/runtime_features.cc

```diff
@@ -61,6 +61,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.conn_pool_delete_when_idle",
     "envoy.reloadable_features.correct_scheme_and_xfp",
     "envoy.reloadable_features.correctly_validate_alpn",
+    "envoy.reloadable_features.defer_processing_backedup_streams",
```

Comment:
Aren't these enabled by default?  The release note says that this behavior is disabled by default.

Reply:
I think we should add a parameter to some of existing test suites. codec_impl_test.cc as well, some integration tests (while adding more shards to avoid it timing out). In this case you do not need to worry about the default value of this flag.